src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4617,7 +4620,9 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 -warnaserror[+|-]             Report all warnings as errors
 -warnaserror[+|-]:&lt;warn list&gt; Report specific warnings as errors
                               (use "nullable" for all nullability warnings)
--warn:&lt;n&gt;                     Set warning level (0-4) (Short form: -w)
```

Comment:
Should we keep the old setting, since still works?

Reply:
No, since we do not want to encourage people to use it. The UI for setting this is also being removed from VS.

---
In reply to: [452492523](https://github.com/dotnet/roslyn/pull/45750#discussion_r452492523) [](ancestors = 452492523)